compositor_render/src/transformations/text_renderer.rs

```diff
@@ -242,15 +250,12 @@ impl TextRendererNode {
             if let Some(layout) = line.layout_opt() {
                 for layout_line in layout {
                     lines_count += 1;
-                    width = max(width, layout_line.w.ceil() as u32);
+                    width = max(width, layout_line.w.ceil() as usize);
                 }
             }
         }
 
-        let height = lines_count * line_height.ceil() as u32;
-        Resolution {
-            width: width as usize,
-            height: height as usize,
-        }
+        let height = (lines_count as f32 * line_height.ceil() + font_size / 4.0) as usize;
```

Comment:
1/4 font_size minimal value? Seems like a lot

Also maybe extract that to separate variable that have a meaningful name e.g
`let last_line_padding = font_size / 4.0`

Reply:
With 1/6 `j` is cut almost always, with 1/5 I didn't find the configuration that cuts `j` or `p`, but I left 1/4 as a margin of error. I'll use 1/5 and change it if we encounter problems in the future.